sirepo/pkcli/admin.py

```diff
@@ -55,20 +80,50 @@ def create_examples():
         with sirepo.auth_db.session_and_lock(), auth.set_user_outside_of_http_request(
             uid
         ):
-            for sim_type in feature_config.cfg().sim_types:
-                names = [
-                    x.name
-                    for x in simulation_db.iterate_simulation_datafiles(
-                        sim_type,
-                        simulation_db.process_simulation_list,
-                        {
-                            "simulation.isExample": True,
-                        },
-                    )
-                ]
-                for example in simulation_db.examples(sim_type):
-                    if example.models.simulation.name not in names:
-                        _create_example(example)
+            callback()
+
+
+def reset_examples():
+    def _reset():
+        c = []
+        r = []
+        for sim_type in feature_config.cfg().sim_types:
```

Comment:
I think the word "sim" appears too many times so labeling this `t` will make it easier to see in the code.